pramen/core/src/main/scala/za/co/absa/pramen/core/rdb/PramenDb.scala

```diff
@@ -77,14 +81,28 @@ class PramenDb(val jdbcConfig: JdbcConfig,
     }
   }
 
+  def addColumn(table: String, columnName: String, columnType: String): Unit = {
+    try {
+      val quotedTable = s""""$table""""
+      val quotedColumnName = s""""$columnName""""
+      db.run(
+          sqlu"ALTER TABLE #$quotedTable ADD #$quotedColumnName #$columnType"
```

Comment:
Adding new column is something that we haven't done in Pramen yet.

We need to test if adding a new column does not break existing pipelines running with the same DB.

I'm 90% sure that adding a new column creates a nullable column by default.

Let me know, I can create a PostgreDB for testing purposes

Reply:
I have already done it and checked with a clone of aqueduct-pramen-boookeeping Aurora DB cluster.
Postgres docs also says that `ALTER TABLE ... ADD` will add nullable column by default.
HSQL also works fine.